src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/versionMatcher/versionObjectAmzn.cpp

```diff
@@ -11,4 +11,4 @@
 
 #include "versionObjectAmzn.hpp"
 
-std::regex VersionObjectAmzn::m_parserRegex(R"((\d+:)?(\d+)\.?(\d+)?\.?(\d+)?-(\d+)?)");
+std::regex VersionObjectAmzn::m_parserRegex(R"((\d+:)?(\d+)\.?(\d+)?\.?(\d+)?-?(\d+)?)");
```

Comment:
Note: Since the group after the dash (`-`) is optional, the dash should be optional as well.

Reply:
The same happens with the previous dots (`.`): `\.?`.